packages/core/admin/index.js

```diff
@@ -229,13 +121,14 @@ async function watchAdmin({ plugins, dir, host, port, browser, options }) {
   const pluginsPath = Object.keys(plugins).map(pluginName => plugins[pluginName].pathToPlugin);
 
   const args = {
-    entry,
+    appDir,
     cacheDir,
-    pluginsPath,
     dest,
+    entry,
     env,
-    port,
     options,
+    pluginsPath,
+    // port,
```

Comment:
Why did we remove the port option?

Reply:
It's an option for the `devServer` and not webpack